Client/game_sa/CWeatherSA.h

```diff
@@ -55,10 +55,15 @@ class CWeatherSA : public CWeather
     bool  SetRainbow(float fAmount);
     bool  ResetRainbow();
 
+    bool SetTimerCycle(unsigned char value) noexcept;
```

Comment:
if it always return true, use `void` instead of `bool`

Reply:
but it's needed later for SetWorldProperty
return g_pGame->GetWeather()->SetTimerCycle(arg1);

Reply:
> but it's needed later for SetWorldProperty return g_pGame->GetWeather()->SetTimerCycle(arg1);

ah. Alright